src/test/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourceRoutesSpec.scala

```diff
@@ -669,6 +669,7 @@ class ResourceRoutesSpec
 
       val endpoints = List(
         s"/v1/files/$organization/$project/nxv:$genUuid?rev=1",
+        s"/v1/files/$organization/$project/nxv:$genUuid?rev=1&output=binary",
```

Comment:
You're breaking the API; you're replacing the content negociation that we've configured with a query param. Revert.